backend/src/test_LLM/LLM_test.py

```diff
@@ -50,42 +52,37 @@ def run_test_instance(log_file_path, questions, excel_path):
 
     wb = openpyxl.Workbook()
     sheet = wb.active
-    sheet.title = "Test result" # byt denna till namnet på test varianten
-    sheet.append(["Question", "Answer"])
-
-    for question in questions:
-        try:
-            last_call_time = time.time()
-            last_call_time = rate_limit(last_call_time)
-            relevant_docs = qachain.retriever.get_relevant_documents(question)
-            retrieved_context = "\n".join(doc.page_content for doc in relevant_docs)
-
-            formatted_prompt = template.format(chat_instruction=chat_instruction,
-                                                                    context=retrieved_context,
-                                                                    query=question)
-            response = qachain.invoke({"query": formatted_prompt})
-            filtered_answer= filter_answer( response['result'])
-            sheet.append([question, filtered_answer])
-            print(f"Q: {question}\nA: {filtered_answer}\n")
-
-        except Exception as e:
-            print("Error: Unexpected error, try again", e)
+    sheet.title = "Test result True Positive" # byt denna till namnet på test varianten
```

Comment:
Kanske sätta upp vad alla olika heter, typ som att man skapar variabler med de godkända strängarna i så det bli enkelt utan probelm

Reply:
https://github.com/logpai/loghub?tab=readme-ov-file

gick in till denna hittade ssh och sen tog jag fram delar av dem som jag tyckte såg ut som brute force